Configure

```diff
@@ -24168,6 +24197,7 @@ d_eofnblk='$d_eofnblk'
 d_erf='$d_erf'
 d_erfc='$d_erfc'
 d_eunice='$d_eunice'
+usedefaultstrict='$usedefaultstrict'
```

Comment:
This should probably move down to keep the list alphabetically ordered?

Reply:
yep. we have a test for this but apparently it doesn't cover Configure.

Reply:
Actually on review of the vars, I see no sorting at all for these variables. I suspect it's going to get clobbered when @Tux adds his module so I'm not sure it's worth doing.

Reply:
It is not. Will be re-ordered later.